gitcheats.md

```diff
@@ -20,45 +21,47 @@ Set your email address:
 >` git config user.email "MY_NAME@example.com" `
 
 Verify your configuration by displaying your configuration file:
->` cat .git/config `
+>**` cat .git/config `**
 
->` git config user.password "your password" ` (optional)
+>**` git config user.password "your password" `** (optional)
 
 <br>
 <br>
 
-### We’ll work with GitHub projects in two ways.
+### Github projects can be done in two ways.
```

Comment:
Ig this will be better - We can work setup GitHub project if following two ways

Reply:
Made changes to the document please check